applications/kodadot_assethub_nft_indexer_statemine_statemint.md

```diff
@@ -0,0 +1,210 @@
+
+
+### Team Code Repos
+
+- [https://github.com/kodadot/stick](https://github.com/kodadot/stick)
+- [https://github.com/kodadot/nft-gallery](https://github.com/kodadot/nft-gallery)
+
+#### Team GitHub accounts 🧑‍💻
+
+- [Viktor Valaštín](https://github.com/vikiival)
+- [Matej Nemček](https://github.com/yangwao)
+
+
+### Team LinkedIn Profiles 🧑‍🎓
+
+- [Matej Nemcek](https://linkedin.com/in/mnemcek/)
+- [Viktor Valastin](https://linkedin.com/in/vikival/)
+
+
+## Development Status :open_book:
+
+- [github.com/kodadot/stick/milestone/1](https://github.com/kodadot/stick/milestone/1) - Milestone 1 track
+- [github.com/kodadot/stick](https://github.com/kodadot/stick) This is the repo for AssetHub Indexer.
+- [github.com/kodadot/uniquery](https://github.com/kodadot/uniquery) This is the repo for SDK.
+
+## Development Roadmap :nut_and_bolt:
+
+## Milestone 1 - AssetHub Indexer Implementation first part
+- **Estimated duration:**  3 months ⌛️
+- **FTE:**  2 FTE
+- **Costs:**  30,000 USD 💰
+
+
+| Sequence | Deliverable                                         | Description                                                                                                   |
+|----------|-----------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| 0a.   | Licensing                                             | MIT License will be applicable.                                                                               |
+| 0b.   | Documentation                                       | Comprehensive inline code documentation and an explicit README file to guide the project setup and execution.|
+| 0c.   | Test Guidelines                                      | Testing will cover major functionality with unit tests and provide a guide for executing these tests.|
+| 0d.   | Docker Integration                                   | A Dockerfile will enable the project to run within a Docker container.                          |
+| 1.      | Backward Compatibility Maintenance             | Ensuring backward compatibility with current Uniques v1.                                                      |
+| 1a.   | Collection Schema Development              | Development of a GraphQL schema entity that represents the collection.                                        |
+| 1b.   | NFT Schema Development                       | Formulation and creation of a GraphQL schema entity representing Non-fungible tokens.                         |
+| 2.      | Forward Support Implementation             | Future support included for NFT pallets comprising soulbound NFTs.                                           |
```

Comment:
What's exactly being done here? What does 'future support' mean?

Reply:
I'm afraid "future support" doesn't fit into the grants program either. You could apply for a maintenance grant to fund ongoing maintenance of the project once it's implemented, but not as part of a regular grant application.

Reply:
Sure, I can remove it.

Reply:
@semuelle this was meant to support soulbond (non-transferable) nfts. 

> Why we used forward?

Because nfts can become sb after certain period of time, but we will change the wording.